src/components/Problems/Discussion.tsx

```diff
@@ -55,9 +58,12 @@ export const Discussion: FC<DiscussionProps> = ({problemId, problemNumber, close
   })
 
   const {mutate: hideComment} = useMutation({
-    mutationFn: (id: number) => axios.post(`/api/competition/comment/${id}/hide`),
+    mutationFn: (id: number) =>
+      axios.post(`/api/competition/comment/${id}/hide`, {hidden_response: hiddenResponseText}),
```

Comment:
idealne by sme passli aj `hiddenResponseText` ako argument mutate-u... tzn. takto:
```suggestion
    mutationFn: (id: number, hidden_response: string) =>
      axios.post(`/api/competition/comment/${id}/hide`, {hidden_response}),
```
a neskor teda:
```ts
hideComment(comment.id, hiddenResponseText)
```